### PR TITLE
update profile documentation page

### PR DESCRIPTION
**Pull request by cpanato**
## Description
While trying to reproduce an issue described https://github.com/openfaas/faas-netes/issues/868 I saw some updates that need to be made in the profile page documentation for some snippets to work

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: docs/reference/profiles.md [hunk 1]
```diff
@@ -118,15 +118,18 @@ For example, a mixture of taints and affinity can put less critical functions on
...
+
     This default installation will enable Profiles.
 
 2. Label _and_ Taint the node with the GPU
+
     ```sh
-    kubectl labels nodes node1 gpu=installed
-    kubectl taint nodes node1 gpu:NoSchedule
+    kubectl label nodes <NODE_NAME> gpu=installed
+    kubectl taint node <NODE_NAME> gpu:NoSchedule
```
Comment (LucasRoesler) [inline docs/reference/profiles.md:132]:
```suggestion
    kubectl taint nodes <NODE_NAME> gpu:NoSchedule
```

the help text for `kubectl taint --help` uses the plural here although i suspect both forms are supported. But using the plural on both lines will make it more consistent.

**Review by LucasRoesler — CHANGES_REQUESTED** [review]

File: docs/reference/profiles.md (hunk 1 above)
Comment (alexellis) [reply docs/reference/profiles.md:132]: If you want to accept this @cpanato, you'll need to manually type in your Signed-off-by line in the commit message.

Screenshot 2021-10-31 at 09 52 47

Comment (cpanato) [reply docs/reference/profiles.md:132]: thanks for the review and suggestions, made the requested change

**Review by LucasRoesler — APPROVED** [review]

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**